varsha/java8features/src/main/java/com/yash/com/java8features/App.java

```diff
@@ -0,0 +1,13 @@
+package com.yash.com.java8features;
```

Comment:
:heavy_multiplication_x: This makes me think you all are copying code from each other. I want you guys to be as independent as possible. Going forward I all ask you to review each others code